docs/extended_api/pipelines/pipeline_consumer_wait_prior.md

```diff
@@ -0,0 +1,64 @@
+---
+grand_parent: Extended API
+parent: Pipelines
+---
+
+# `cuda::pipeline_consumer_wait_prior`
+
+```c++
+// Header `<cuda/pipeline>`.
```

Comment:
Same comment about "declared in" style as earlier.

This file needs a comment that this functionality is not a part of the github-released open-source library.